src/shipit_code_coverage/shipit_code_coverage/artifacts.py

```diff
@@ -84,6 +84,10 @@ def download_all(self):
         for test_task in test_tasks:
             status = test_task['status']['state']
             assert status in ALL_STATUSES, "State '{}' not recognized".format(status)
+            # Skip tasks that will not have artifacts ready, like the ones that
```

Comment:
This should never happen:
- the normal coverage job is triggered by a pulse message that is sent when all the tasks in the group are finished;
- the periodic (chunk mapping) coverage job is using the revision of the latest successful normal coverage job.